.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,59 @@
+name: Run Unit Tests
+
+on: [push, pull_request]
+
+jobs:
+  unit-test:
+    runs-on: ubuntu-20.04
+    services:
+      mariadb:
+        image: mariadb:10.4
+        env:
+          MARIADB_ALLOW_EMPTY_ROOT_PASSWORD: 'yes'
+          MARIADB_DATABASE: apel_unittest
+        ports:
+          - 3306:3306
+        options: >-
```

Comment:
Can you add some comments to explain these options, please?

Reply:
These options are docker container options to see the container health status. Not a dependency thou to successfully run the testcases. But, if we provide health checks, the container will not start until the db container is healthy (basically until mysqladmin is up and accepting connections).

mysqladmin ping will return a false positive if the server is running but not yet accepting connections.


Just to summarise from the link below:

`--health-cmd	        Command to run to check health `
`--health-interval	Time between running the check`
`--health-retries	Consecutive failures needed to report unhealthy`
`--health-timeout	Maximum time to allow one check to run\n`



For further information. Please refer to https://docs.docker.com/engine/reference/run/#healthchecks

Reply:
Can you add the summarised comments to the code please?

Reply:
Sure Adrian. Added this comments into the code now. [69c7119](https://github.com/apel/apel/pull/353/commits/69c71195d08fb9420eef3358e50f21fe4cadb263)